src/mcts/search.cc

```diff
@@ -1190,7 +1190,7 @@ void SearchWorker::FetchSingleNodeResult(NodeToProcess* node_to_process,
       // Flush denormals to zero.
       p = p < 1.17549435E-38
               ? 0.0
-              : FastPow2(FastLog2(p) / params_.GetPolicySoftmaxTemp());
+              : FastPow2(FastLog2(p) / (params_.GetPolicySoftmaxTemp() + FastLog2(node->GetN()+1) ));
```

Comment:
I don't think this does what you think - this is only called once for a fresh node and then never again - so its GetN should always be 0.